guides/source/active_support_core_extensions.md

```diff
@@ -837,6 +839,25 @@ In the previous example the macro generates `avatar_size` rather than `size`.
 
 NOTE: Defined in `active_support/core_ext/module/delegation.rb`
 
+#### `delegate_missing_to`
+
+Imagine you would like to delegate all of the `Profile` methods to the `User`
+object. Instead of listing every `Profile` method in a `delegate` call, you can
```

Comment:
No, because that still doesn't match what the code says: `User.delegate_missing_to :profile`